schema/system.hcl

```diff
@@ -1,3 +1,41 @@
+table "access_tokens" {
+  schema = schema.public
+  column "id" {
+    null    = false
+    type    = uuid
+    default = sql("generate_ulid()")
+  }
+  column "person_id" {
+    null = false
+    type = uuid
+  }
+  column "value" {
+    null = false
+    type = text
+  }
+  column "created_at" {
+    null = false
+    type = timestamptz
+  }
+  column "expires_at" {
+    null = true # We can have never expiring tokens
```

Comment:
Should we have never expiring tokens? @moshloop

Reply:
I think we can handle this later